client/commands/cutoff.js

```diff
@@ -56,7 +56,7 @@ const requestRate = () => {
  * Calculates the standard error of 
  * @param {Array} Array of all data points
  * @param {Object} Linear Regression Model
- * @param {Integer} Calculated Final Rate of the event
```

Comment:
This is still part of the input parameters right? Not sure if you want to remove this.